packages/listen-window-errors/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "build"
```

Comment:
I see these `compilerOptions` all spread around, can't we keep it on the main `tsconfig.json`?